packages/block-editor/src/components/list-view/branch.js

```diff
@@ -133,15 +135,15 @@ function ListViewBranch( props ) {
 	const blockCount = filteredBlocks.length;
 	// The appender means an extra row in List View, so add 1 to the row count.
 	const rowCount = showAppender ? blockCount + 1 : blockCount;
-	let nextPosition = listPosition;
+	nextPositionRef.current = listPosition;
```

Comment:
This probably goes against the "don't read/write refs during render" rule, no?

Reply:
I thought it might, but it doesn't really trigger an error. And I think we can find similar usage across the repo already, so 🤷

Reply:
I wonder why exactly the compiler didn't like the `nextPosition` local variable. Maybe it's that it's incremented in a `map` callback and the compiler thinks that the callback is an event handler?

`nextPosition` is a legit local variable, assigned and used only during the render, and the value doesn't need to survive the render function. The ref is not really needed. Reading/writing it is safe because, well, it's used as a local variable.